<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialWidget</class>
 <widget class="QWidget" name="MaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
    border: 1px solid gray;
    border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    margin-top: 2ex;
}

QGroupBox::title
{
    background: #bbb;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 2px 800px 2px 3px;
}

QSlider
{
    height: 18px;
}

QSlider::groove:horizontal
{
    border: 1px solid #999999;
    height: 2px;
}

QSlider::handle:horizontal
{
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 10px;
    margin: -5px 0;
    border-radius: 5px;
}

QRadioButton
{
    color: black;
    height: 14px;
}

QRadioButton::indicator
{
    color: black;
    width: 10px;
    height: 10px;
    top: 1;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,1">
   <item>
    <widget class="QGroupBox" name="Shader">
     <property name="title">
      <string>Shader</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QComboBox" name="ShaderList">
          <item>
           <property name="text">
            <string>Ocean</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Albedo">
     <property name="title">
      <string>Albedo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="Color">
          <property name="styleSheet">
           <string notr="true">QGroupBox::title
{
    background: none;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px 0 3px;
    left: 8px;
}</string>
          </property>
          <property name="title">
           <string>Colour</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="topMargin">
            <number>14</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="ShallowColor">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
   background-color: white;
   border: 1px solid black;
}</string>
               </property>
               <property name="text">
                <string>Shallow</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DeepColor">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
   background-color: white;
   border: 1px solid black;
}</string>
               </property>
               <property name="text">
                <string>Deep</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FresnelColor">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
   background-color: white;
   border: 1px solid black;
}</string>
               </property>
               <property name="text">
                <string>Fresnel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>47</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QToolButton" name="ResetAlbedo">
                 <property name="text">
                  <string>@</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../material/materialplugin.qrc">
                   <normaloff>:/materialplugin/reset.png</normaloff>:/materialplugin/reset.png</iconset>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="DepthScaleLabel">
               <property name="text">
                <string>Depth Scale :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QcDoubleSpinBox" name="DepthScale">
               <property name="minimumSize">
                <size>
                 <width>52</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="DepthScaleLabel2">
               <property name="minimumSize">
                <size>
                 <width>8</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Roughness">
          <property name="styleSheet">
           <string notr="true">QGroupBox::title
{
    background: none;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px 0 3px;
    left: 8px;
}</string>
          </property>
          <property name="title">
           <string>Roughness</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="topMargin">
            <number>14</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QcDoubleSlider" name="RoughnessSlider">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QcDoubleSpinBox" name="RoughnessSpinner">
               <property name="minimumSize">
                <size>
                 <width>52</width>
                 <height>0</height>
                </size>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RoughnessLabel">
               <property name="minimumSize">
                <size>
                 <width>8</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Reflectivity">
          <property name="styleSheet">
           <string notr="true">QGroupBox::title
{
    background: none;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px 0 3px;
    left: 8px;
}</string>
          </property>
          <property name="title">
           <string>Reflectivity</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="topMargin">
            <number>14</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QcDoubleSlider" name="ReflectivitySlider">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QcDoubleSpinBox" name="ReflectivitySpinner">
               <property name="minimumSize">
                <size>
                 <width>52</width>
                 <height>0</height>
                </size>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ReflectivityLabel">
               <property name="minimumSize">
                <size>
                 <width>8</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Emissive">
          <property name="styleSheet">
           <string notr="true">QGroupBox::title
{
    background: none;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px 0 3px;
    left: 8px;
}</string>
          </property>
          <property name="title">
           <string>Emissive</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <property name="topMargin">
            <number>14</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QcDoubleSlider" name="EmissiveSlider">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QcDoubleSpinBox" name="EmissiveSpinner">
               <property name="minimumSize">
                <size>
                 <width>52</width>
                 <height>0</height>
                </size>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="EmisionLabel">
               <property name="minimumSize">
                <size>
                 <width>8</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="AlbedoOutput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>rgba</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Normal">
     <property name="title">
      <string>Normal</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,0">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="DropLabel" name="NormalMap">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>normal</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QToolButton" name="ResetNormal">
              <property name="text">
               <string>@</string>
              </property>
              <property name="icon">
               <iconset resource="../material/materialplugin.qrc">
                <normaloff>:/materialplugin/reset.png</normaloff>:/materialplugin/reset.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="NormalOutput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>xyz</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_16">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Surface">
     <property name="title">
      <string>Surface</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="DropLabel" name="SurfaceMap">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>foam</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QToolButton" name="ResetSurface">
              <property name="text">
               <string>@</string>
              </property>
              <property name="icon">
               <iconset resource="../material/materialplugin.qrc">
                <normaloff>:/materialplugin/reset.png</normaloff>:/materialplugin/reset.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="SurfaceOutput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>rgba</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>DropLabel</class>
   <extends>QLabel</extends>
   <header>droplabel.h</header>
  </customwidget>
  <customwidget>
   <class>QcDoubleSlider</class>
   <extends>QSlider</extends>
   <header>qcdoubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../material/materialplugin.qrc"/>
 </resources>
 <connections/>
</ui>
